docs/source/contributor-guide/development.md

```diff
@@ -85,6 +85,32 @@ mvn test -Dsuites="org.apache.comet.CometCastSuite valid" -Dskip.surefire.tests=
 
 Other options for selecting specific suites are described in the [ScalaTest Maven Plugin documentation](https://www.scalatest.org/user_guide/using_the_scalatest_maven_plugin)
 
+## Plan Stability Testing
+
+Comet has a plan stability testing framework that can be used to test the stability of the query plans generated by Comet.
+The plan stability testing framework is located in the `spark` module and can be run using the following command:
+
+```
+mvn -pl spark -Dsuites="org.apache.spark.sql.comet.CometTPCDSV1_4_PlanStabilitySuite" test
+```
+
+and
+```
+mvn -pl spark -Dsuites="org.apache.spark.sql.comet.CometTPCDSV2_7_PlanStabilitySuite" test
+```
+
+If your pull request changes the query plans generated by Comet, you should run the plan stability tests to regenerate the golden files.
```

Comment:
nit: I guess we can remove `run the plan stability tests to`

Reply:
Should we have a CI check that runs the plan stability tests to generate the golden files and then fail the build if the generated files cause a local diff?

Reply:
> nit: I guess we can remove `run the plan stability tests to`

Okay.